datafusion/src/logical_plan/expr.rs

```diff
@@ -974,7 +974,7 @@ impl std::fmt::Display for Expr {
                 ref left,
                 ref right,
                 ref op,
-            } => write!(f, "{} {} {}", left, op, right),
+            } => write!(f, "{:?} {} {:?}", left, op, right),
```

Comment:
Why the change to use `Debug` formatting rather than `Display` formatting ?

Reply:
Ah, this was part of my own debugging when trying to sort out why my queries were failing. I'll remove.